### PR TITLE
Add the ability to configure access_management capabilities

### PR DESCRIPTION
**Pull request by fuziontech**
## Description
We have a paying user requesting to have the ability to create users dynamically using SQL
https://clickhouse.com/docs/en/sql-reference/statements/create/user/

In order to do this we must enable `access_management` in the `users.xml` configs.
https://clickhouse.com/docs/en/operations/settings/settings-users#access_management-user-setting

This PR makes it possible to enable this through the following values:
```yaml
clickhouse:
  enable: true
  additionalUsersConfig:
    admin/access_management: "1"
```

Requested by customer here:
https://posthog.slack.com/archives/C03G21222QY/p1668109138746399?thread_ts=1666628600.482729&cid=C03G21222QY

## Type of change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## How has this been tested?
Tested on a Digitalocean K8s cluster 

## Checklist
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: charts/posthog/templates/clickhouse_instance.yaml [hunk 1]
```diff
@@ -17,6 +17,9 @@ spec:
       {{ .Values.clickhouse.backup.backup_user }}/networks/ip: "0.0.0.0/0"
       {{ template "clickhouse.backupPasswordValue" . }}
       {{- end}}
+      {{- if .Values.clickhouse.users }}
```
Comment (guidoiaquinti) [inline charts/posthog/templates/clickhouse_instance.yaml:20]: I find this a bit confusing as we have already other values (like: `clickhouse.user` defined). 

What about `additionalUsersConfig`?

File: charts/posthog/values.yaml [hunk 2]
```diff
@@ -39,12 +39,10 @@ env: []
 #   - name: FOO
 #     value: bar
 
-
```
Comment (guidoiaquinti) [inline charts/posthog/values.yaml:42]: Lots of formatting changes were committed unrelated to this change? Not sure if this was expected.

Comment (fuziontech) [reply charts/posthog/values.yaml:42]: It was. My IDE automatically formats yaml. Seems like most of this makes sense to me.

**Review by hazzadous — APPROVED** [review]

**Comment by hazzadous** [comment]
Have made a format updates as well as including the new value in the values.yaml cc @fuziontech @guidoiaquinti 😓

**Review by guidoiaquinti — APPROVED** [review]

**Pull request merged**